### DISCUSSION TITLE
Rename Does not work for some payment method even payment method has no logo

### DISCUSSION THREAD

**Discussion by siddharth222** [Shopify/function-examples#245] [Feedback: Payment Customization API]
Ex: i tried for PayPlus https://prnt.sc/VQU0msjepQq9 rename works fine on cargo test but Shopify do not change it!

**Comment by siddharth222** [comment]
Nets Easy Checkout
ViaBill Payments
Mobilepay via Quickpay

above payment also could not rename! any idea..

**Comment by nickwesselman** [comment]
Renames may not work consistently if the merchant is not on the new checkout experience. I would recommend that if the rename isn’t working, that the merchant reaches out to merchant support and ask what is preventing their shop from updating to the new checkout experience.

**Reply by siddharth222** [reply]
Got it.. 
Thanks for update :)
